gptme/tools/base.py

```diff
@@ -56,12 +96,15 @@ class ToolSpec:
     name: str
     desc: str
     instructions: str = ""
+    function_description: str = ""
```

Comment:
I think we should somehow merge this with the already existing `instructions` property.

In the cases where `instructions` contain format-specific instructions, we could move such instructions into format-specific prompting under a new `instructions_format` dict with keys like `markdown`.

Reply:
Good idea. Done.